File: ruby/ebeats.rb
#!/usr/bin/env ruby

time = Time.now.utc.to_a
ratio = Rational(1000, 24 * 60 * 60), Rational(1000, 24 * 60), Rational(1000, 24)

puts "@%.2f" % time[0,3].zip(ratio).map{|t,r| t * r }.inject(:+)

File: go/ebeats.go
// time in ebeats
package main

import (
	"fmt"
	"time"
)

func main() {
	h, m, s := time.Now().UTC().Clock()
	fmt.Printf("@%.2f\n", 5*(60*(60*float64(h)+float64(m))+float64(s))/432)
}
